src/client/utils/dom.js

```diff
@@ -636,15 +637,22 @@ export function isPerformanceNavigationTiming (entry) {
     return instanceToString(entry) === '[object PerformanceNavigationTiming]';
 }
 
-export function isArrayBufferData (array) {
-    if (array[0] instanceof ArrayBuffer)
+export function isArrayBuffer (data) {
+    if (data instanceof ArrayBuffer)
```

Comment:
All constructors (`ArrayBuffer`, `DataView`) should be stored in `nativeMethods` and used from it